File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/InvalidWalletException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when an attempt is made to use a closed or invalid wallet.
    /// </summary>
    public class InvalidWalletException : IndyException
    {
        const string message = "The wallet is closed or invalid and cannot be used.";

        /// <summary>
        /// Initializes a new WalletClosedException.
        /// </summary>
        internal InvalidWalletException() : base(message, (int)ErrorCode.WalletInvalidHandle)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/wallet/InvalidWalletException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when attempting to use a wallet that has been closed.
 */
public class InvalidWalletException extends IndyException
{
	private static final long serialVersionUID = -606730416804502147L;
	private final static String message = "The wallet is closed or invalid and cannot be used.";

	/**
	 * Initializes a new WalletClosedException.
	 */
	public InvalidWalletException() {
		super(message, ErrorCode.WalletInvalidHandle.value());
	}
}
